calcipy/noxfile/_noxfile.py

```diff
@@ -1,17 +1,17 @@
-"""nox-poetry configuration file.
+"""nox-uv configuration file.
 
 [Useful snippets from docs](https://nox.thea.codes/en/stable/usage.html)
 
 ```sh
-poetry run nox -l
-poetry run nox --list-sessions
+uv run nox -l
+uv run nox --list-sessions
 
-poetry run nox -s build_check-3.8 build_dist-3.8 tests-3.8
-poetry run nox --session tests-3.11
+uv run nox -s build_check-3.8 build_dist-3.8 tests-3.8
+uv run nox --session tests-3.11
 
-poetry run nox --python 3.8
+uv run nox --python 3.8
 
-poetry run nox -k "not build_check and not build_dist"
+uv run nox -k "not build_check and not build_dist"
```

Comment:
## poetry run equivalent / venv discovery

With `uv`, there doesn't appear to be an equivalent of `poetry run`, so you likely need to find the `.venv/bin/python` or call `source .venv/bin/activate` (Mac) or `.venv\Scripts\activate` (Windows)

See: https://docs.astral.sh/uv/pip/environments/#discovery-of-python-environments

Calcipy could do this generically by iterating through `os.getenv("VIRTUAL_ENV", os.getenv("CONDA_PREFIX")) or <search for ./.venv>`

**Update**: this does actually seem to be possible https://github.com/charliermarsh